test/types/chplhashtable/map-capacity.good

```diff
@@ -0,0 +1,8 @@
+Map 1 capacity: 0.5
+Map 2 capacity: 0.5
+Map 1 capacity: 0.265625
+Map 2 capacity: 0.53125
+Map 1 capacity: 0.375
+Map 2 capacity: 0.75
+Map 1 capacity: 0.390625
+Map 2 capacity: 0.390625
```

Comment:
Could you add two more tests for error messages for when you have `resizeThreshold==0` and `1` ?